benchexec/tools/vampire.py

```diff
@@ -32,15 +32,15 @@ def environment(self, executable):
 
     def cmdline(self, executable, options, task, rlimits):
         if "-t" not in options and "--time_limit" not in options:
-            if rlimits.walltime:
```

Comment:
I would suggest the following alternatives (in order of preference) as these would probably make the code better understandable in the future:

1. Remove branching completely and use `options += ["-t", f"{rlimits.walltime or 0}s"]` (with explaining comment). This is the usual way for fallback values in Python and avoids the duplicate code.
2. Swapping branches instead of changing condition.
3. Use `if not rlimits.walltime:` (more "Pythonic" than comparison with `None`).